src/AutoBalance.cpp

```diff
@@ -709,8 +709,8 @@ class AutoBalance_AllCreatureScript : public AllCreatureScript
             else {
                 newDmgBase=creatureStats->BaseDamage[2];
                 // special increasing for end-game contents
-                if (LevelEndGameBoost && !creature->GetMap()->IsRaid())
-                    newDmgBase *= creatureABInfo->selectedLevel >= 75 && originalLevel < 75 ? float(creatureABInfo->selectedLevel-70) * 0.3f : 1;
+            if (LevelEndGameBoost && !creature->GetMap()->IsRaid())
+                newDmgBase *= creatureABInfo->selectedLevel >= 75 && originalLevel < 75 ? float(creatureABInfo->selectedLevel-70) * 0.3f : 1;
```

Comment:
add curly brackets

Reply:
@Helias also got this one done. Let me know if that looks good now.